addons/base_import/static/src/js/import.js

```diff
@@ -405,7 +405,7 @@ var DataImport = Widget.extend(ControlPanelMixin, {
                 minimumInputLength: 0,
                 data: filtered_data,
                 initSelection: function (element, callback) {
-                    var default_value = element.val();
+                    var default_value = (element.val() || '').split('/id')[0];
```

Comment:
Thanks, indeed it works by activating "Show all fields for completion (advanced)". Although I don't really get the point not recognizing some fields by default.